public/styles/index.css

```diff
@@ -127,6 +139,32 @@
     .__footerp {
         margin-top: 30px;
     }
+    .__shecodeheader_text {
+        margin-top: 0!important;
+    }
+    .__shecodeheader_title h1 {
+        font-size: 40px!important;
+    }
+    .__shecodeheader_subtitle h4 {
+        font-size: 18px!important;
+        margin-top: 20px!important;
+    }
```

Comment:
Here's another set of `!important`s (are they really necessary?) and headers with different sizes (which should ideally be uniform)